issue_template.md

```diff
@@ -0,0 +1,18 @@
+**Description:**
+<!-- Give a brief description of the issue -->
+
+**Suggested Labels:**
+<!-- Optional comma separated list of suggested labels. Non committers can’t assign labels to issues, so this will help issue creators who are not a committer to suggest possible labels-->
+
+**Suggested Assignees:**
```

Comment:
Shall we move this to bottom, and move **OS, DB, other environment details and versions:**   and **Steps to reproduce:** to top.